jumpscale/clients/taiga/taiga.py

```diff
@@ -217,3 +217,137 @@ def _get_project_status(project_statuses, status):
 
         self.api.user_stories.delete(story_id)
         return migrate_story.id
+
+    def list_all_issues(self, user_id=""):
+        """
+        List all issues for specific user if you didn't pass user_id will list all the issues
+        
+        Args:
+            user_id (int): id of the user.
+
+        Returns:
+            List: List of issues.
+        """
+        return self.api.issues.list(assigned_to=user_id)
+
+    def list_all_projects(self):
+        """
+        List all projects
+        
+        Returns:
+            List: List of projects.
+        """
+        return self.api.projects.list()
+
+    def list_all_milestones(self):
+        """
+        List all milestones
+        
+        Returns:
+            List: List of milestones.
+        """
+        return self.api.milestone.list()
+
+    def list_all_user_stories(self, user_id=""):
+        """
+        List all user stories for specific user if you didn't pass user_id will list all the available user stories
+        
+        Args:
+            user_id (int): id of the user.
+            
+        Returns:
+            List: List of user stories.
+        """
+        return self.api.user_stories.list(assigned_to=user_id)
+
+    def get_issues_export_text(self, prepend_text="Issues", issues=None):
+        """Get issues subject and append them to text
+        
+        Args:
+            prepend_text (str): the text that the method will append to it.
+            issues (List): list of all issues that we will get the subject from .
+            
+        Returns:
+            str: string contains the issues subject.
+        """
+        for issue in issues:
+            prepend_text += f"- {issue.subject} \n"
+        return prepend_text
+
+    def export_all_issues_subjects(self, path="/tmp/issues_titles.md"):
+        """Export all the issues subjects in a markdown file
+        
+        Args:
+            path (str): The path of exported markdown file.
+            
+        """
+        text = f"""
+### Issues \n
+"""
+        issues = self.list_all_issues()
+        text = self.get_issues_export_text(prepend_text=text, issues=issues)
+
+        j.sals.fs.write_file(path=path, data=text)
+
+    def export_issues_per_project(self, path="/tmp/issues_per_project.md"):
+        """Export issues per project in a markdown file
+        
+        Args:
+            path (str): The path of exported markdown file.
+            
+        """
+        projects = self.list_all_projects()
+        text = """
+### Issues  Per Project \n
+"""
+        for project in projects:
```

Comment:
can we make SINGLE_PROJECT_TEMPLATE variable and use it here instead of writing the data on the go?

Reply:
I am not writing the data on the fly, I do that in the last line of the method 
![image](https://user-images.githubusercontent.com/46320202/95061446-e98fb780-06fb-11eb-9f9b-046c9cb9a290.png)